Makefile.standard_app

```diff
@@ -0,0 +1,160 @@
+#*******************************************************************************
+#   Ledger SDK
+#   (c) 2022 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
+
+#####################################################################
+#                            BLUETOOTH                              #
+#####################################################################
+ifeq ($(ENABLE_BLUETOOTH), 1)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOX TARGET_STAX))
+    STANDARD_APP_FLAGS = 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
+    DEFINES += HAVE_BLE BLE_COMMAND_TIMEOUT_MS=2000 HAVE_BLE_APDU
+    SDK_SOURCE_PATH += lib_blewbxx lib_blewbxx_impl
+endif
+endif
```

Comment:
Should be merged with `DEFINES += BLE_SEGMENT_SIZE=32` below?

Reply:
Dont you think people could enable bluetooth with a custom segment size ?

Reply:
If that happen, they could do it by setting `ENABLE_BLUETOOTH=0` and putting what's needed in their app Makefile?
I guess that should be rare enough to be ok with that?